src/components/pages/Settings/ArNSSettings.tsx

```diff
@@ -45,7 +45,7 @@ function ArNSSettings() {
           processId: id.trim(),
           ao: aoClient,
         }),
-        ...(wallet?.arconnectSigner ? { signer: wallet.arconnectSigner } : {}),
+        ...(wallet?.contractSigner ? { signer: wallet.contractSigner } : {}),
```

Comment:
signer is supposed to be able to be undefined, whats up with this?

Reply:
Unsure, the code was already here that way with arconnectSigner and I modified for the field name change. Looks like there is no overload that takes in both AoProcess and a signer (there's some that uses processId + signer). I assume this works as before as all I did was change the fields, but it seems like something is off here.